src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/SqlConnection.cs

```diff
@@ -1414,6 +1414,12 @@ private async Task ReconnectAsync(int timeout)
                         if (attempt == retryCount - 1)
                         {
                             SqlClientEventSource.Log.TryTraceEvent("SqlConnection.ReconnectAsync | Info | Original Client Connection Id {0}, give up reconnection", _originalConnectionId);
+                            if (e.Class >= TdsEnums.FATAL_ERROR_CLASS)
+                            {
+                                SqlClientEventSource.Log.TryTraceEvent("<sc.SqlConnection.ReconnectAsync|INFO> Original ClientConnectionID {0} - Fatal Error occured. Error Class: {1}", _originalConnectionId, e.Class);
+                                // Errors 20-25, usuallyterminate the database connection
```

Comment:
```suggestion
                                // Error Class: 20-25, usually terminates the database connection
```
NIT: fix typo

Reply:
well, the comment means errors with class numbers in range of 20-25 usually terminate database connections.